README.md

```diff
@@ -50,6 +50,7 @@ A curated list of Minio community projects inspired by [awesome-go](https://gith
 * [Trustvox](http://site.trustvox.com.br) - Trustvox is an integrator of ecommerce platforms, ensures 100% valid product reviews, allowing only real buyers to post reviews along with a continuous audit process. The platform team chose Minio to create its own S3 service to avoid infrastructure vendor lock-in, obtaining transparent compatibility with Google Cloud Storage and previous Amazon S3 infrastructures. We tested other options before, and only Minio showed the stable and enterprise-grade results we were looking for.
 * [www.houbank.com](https://www.houbank.com/home/) - Houbank uses Minio as a simple, reliable and compatible self-host AWS S3 alternative. Houbank uses Minio to upload user files and exchange documents in internal applications/jobs.
 * [Rocket.Chat](https://rocket.chat/) - Rocket.Chat is a completely FOSS alternative to Slack. In Rocket.Chat Minio can be used for file uploads — this applies to channels, groups and privately between one user and another.
+* [Plutify](http://plutify.com/) - Plutify offers cloud storage space to store and serve client files. It also offers services like video sharing, file selling, and direct link. Plutify uses Minio Object Storage as the backend storage platform.
```

Comment:
Maybe just `Plutify offers hosted cloud storage for files and simple sharing via direct links. It uses Minio Object storage as the storage platform`. The video sharing/file selling feature does not seem to be advertised on the website, however if its on good authority we can add it.

Reply:
agreed, I added it as it was in the [original PR](https://github.com/minio/awesome-minio/pull/65), but the site doesn't seem to be offering those things.